### PR TITLE
qa_crowbarsetup: add s390 repos

### PR DESCRIPTION
**Pull request by bmwiedemann**
only when requested with want_s390=1

### CODE REVIEW

**Comment by jdsn** [comment]
:+1:

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1000,6 +1000,10 @@ function onadmin_prepare_sles12sp1_other_repos()
     for repo in SLES12-SP1-{Pool,Updates}; do
         add_mount "$repo/sle-12-x86_64" "$clouddata:/srv/nfs/repos/$repo" \
             "$tftpboot_repos12sp1_dir/$repo"
+        if [[ $want_s390 ]] ; then
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: hmm, so want_s390=0 would be true..

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: yes, but we have that same behaviour in many other places as well, where we test  with [ -n $boolean ] or [[ ]]

**Comment by toabctl** [comment]
+1 . Let's fix the bool check in another PR..

**Pull request merged**